src/login/login.component.tsx

```diff
@@ -73,14 +75,17 @@ export default function Login(props: LoginProps) {
     return null;
   }
 
+  const logo = config.logoImgSrc ? (
+    <img src={config.logoImgSrc} alt="Partners in Health" />
```

Comment:
The alt should probably come from the config, too, instead of hard coded to PIH.

Reply:
Whoops!

Yes, it probably should, shouldn't it  :laughing: